kong/plugins/moesif/moesif_ser.lua

```diff
@@ -119,6 +119,11 @@ function parse_body(headers, body, mask_fields, conf)
   return body_entity, body_transfer_encoding
 end
 
+local function fetchRouteAndServices(table, fieldName)
```

Comment:
should it be fetchField or something like that.. I don't see anywhere if it is just related to route and service?